gatsby-node.js

```diff
@@ -1,5 +1,7 @@
 const path = require('path')
```

Comment:
What is the purpose of the above `uploaded-images` and `missing-images` files?

Reply:
These were some processed output from the initial import - I just wanted them as a reference in case I messed it up.